src/jit/morph.cpp

```diff
@@ -18154,6 +18127,14 @@ class LocalAddressVisitor final : public GenTreeVisitor<LocalAddressVisitor>
             MorphStructField(node, user);
         }
         else if (node->OperIs(GT_LCL_VAR, GT_LCL_FLD))
+        {
+            if (node->OperIs(GT_LCL_FLD))
```

Comment:
You can modify the `else if` above and remove this one.

Reply:
I though I had removed it. Maybe somehow missed during rebase etc.

Thanks, fixed!